packages/react-core/src/components/Modal/ModalContent.tsx

```diff
@@ -120,10 +126,17 @@ export const ModalContent: React.FunctionComponent<ModalContentProps> = ({
     actions.length > 0 && <ModalBoxFooter>{actions}</ModalBoxFooter>
   );
 
+  const defaultModalBodyAriaRole = bodyAriaLabel ? 'region' : undefined;
+
   const modalBody = hasNoBodyWrapper ? (
     children
   ) : (
-    <ModalBoxBody {...props} {...(!description && !ariaDescribedby && { id: descriptorId })}>
+    <ModalBoxBody
+      aria-label={bodyAriaLabel}
```

Comment:
Should the aria-label have a default?

Reply:
I'm not sure. If it should what would you want the default to be?

Reply:
Hmm this is an interesting question and actually makes me also question the default role. If we're going to default the role then I think we should default an `aria-label`, but do we always want the content of a model to have a role and an `aria-label`? When we were talking about it before, I admittedly was just thinking about the overflow example, but I'm not sure if it always makes sense to have both? For example, it seems a little heavy handed if the modal content is a sentence of information, and we have it segmented out with a role and label.

Reply:
I'm not opposed to adding a default label at all, I just have no idea what a good default would be. I think that any good label here is going to be highly variable based on what the content of the modal is.

Combined with the fact that we currently have the modal itself labeled by the modal heading, and described by the modal body, I'm not sure if having a default would be for the best? I'm not an expert on this topic by any means though and am happy to proceed however you all think is best on it 🙂

Reply:
I will defer to @jessiehuff here.
Jessie, are you saying that the role should only default to `region` if there is an overflow?  Should this props be tied to that having overflow?

Reply:
I think I like Austin's approach of defaulting the region when there's an `aria-label` to prevent a11y issues, but I'm leaning towards not having a default `aria-label`. The only case that I can see that being helpful is in the case of the overflow so perhaps it's better to just allow users to customize it when they'd like.

Reply:
it might be beneficial to mention in the prop description when it should be used?